backend/service/service_assets.go

```diff
@@ -221,13 +221,13 @@ func convertModelActualFee(actfee document.ActualFee) vo.ActualFee {
 func (service *AssetsService) UpdateAssetTokens(vs []document.Asset) error {
```

Comment:
this method should remove asset token `iris`, there is no need to save and  update ` iris` token info